<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QIMUWidget</class>
 <widget class="QWidget" name="QIMUWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>684</width>
    <height>447</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>300</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>300</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QWidget" name="widget" native="true"/>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0">
     <property name="spacing">
      <number>10</number>
     </property>
     <property name="sizeConstraint">
      <enum>QLayout::SetNoConstraint</enum>
     </property>
     <property name="leftMargin">
      <number>2</number>
     </property>
     <property name="topMargin">
      <number>2</number>
     </property>
     <property name="rightMargin">
      <number>2</number>
     </property>
     <property name="bottomMargin">
      <number>2</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0,0,1">
        <property name="spacing">
         <number>2</number>
        </property>
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="topMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <property name="bottomMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QPushButton" name="rawButton">
          <property name="text">
           <string>raw</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">imuButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ahrsButton">
          <property name="text">
           <string>AHRS</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">imuButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="calibrateButton">
          <property name="text">
           <string>calibrate</string>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">imuButtonGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="plot_stackedWidget">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="imu">
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRunningPlot" name="imu_plot1" native="true"/>
         </item>
         <item>
          <widget class="QRunningPlot" name="imu_plot2" native="true"/>
         </item>
         <item>
          <widget class="QRunningPlot" name="imu_plot3" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="ahrs">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>6</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QRunningPlot" name="ahrs_plot1" native="true"/>
         </item>
         <item>
          <widget class="QRunningPlot" name="ahrs_plot2" native="true"/>
         </item>
         <item>
          <widget class="QRunningPlot" name="ahrs_plot3" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="calib">
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QIMUCalibrationWidget" name="calib_widget" native="true"/>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QRunningPlot</class>
   <extends>QWidget</extends>
   <header>qrunningplot.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QIMUCalibrationWidget</class>
   <extends>QWidget</extends>
   <header>qimucalibrationwidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="imuButtonGroup"/>
 </buttongroups>
</ui>
